uaclient/security_status.py

```diff
@@ -73,13 +73,11 @@ def filter_security_updates(
     ]
 
     return [
-        package
+        version
```

Comment:
I think we need to modify the or unittest to better cover this change, right ?

Reply:
Hmm I was thinking about this.
Since we cannot use the (mocked) `apt` classes in unittests, this would add a bigger complexity to the package mocks.
I will check for a real package and add an integration test targeting it.

Reply:
I mean, forget what I said:
the unittest showed as the best option to cover this. Updated the test steps with an example.